spec/javascripts/spec/track_click.spec.js

```diff
@@ -61,7 +62,24 @@ describe('A click tracker', function() {
       spyOn(root.GOVUKAdmin, 'trackEvent');
       tracker.start(element);
       element.find("a").click();
-      expect(GOVUKAdmin.trackEvent).toHaveBeenCalledWith('a-press', 'bar');
+      expect(GOVUKAdmin.trackEvent).toHaveBeenCalledWith('a-press', 'bar', undefined, undefined);
+    });
+  });
+
+  describe('with action, label, value and category specified', function(){
+    beforeEach(function() {
+      element = $('\
+        <div data-track-action="delete-button-click" data-track-category="userInteraction:LLM">\
+          <a class="js-track" data-track-label="delete-link" data-track-value="link1" >Foo</a>\
```

Comment:
A value must be numeric: https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#eventValue